src/ocamlorg_frontend/pages/learn.eml

```diff
@@ -22,35 +22,38 @@ Learn_layout.single_column_layout
         <%s!  Learn_components.hero_button ~left_icon:(Icons.ocaml "w-5 h-5") ~right_icon:(Icons.download "w-5 h-5") ~text:("Install OCaml " ^ latest_version) ~href:(Url.install) %>
         <div class="flex flex-col gap-4">
           <%s!  Learn_components.hero_button ~left_icon:(Icons.book "w-5 h-5") ~right_icon:(Icons.link "w-5 h-5") ~text:("Standard Library API") ~href:(Url.api) %>
-          <div class="text-sm text-text_title">
+          <div class="text-sm text-text_content dark:text-grey_navigation">
             The index of types, exceptions, values, modules, and module types of the OCaml Standard Library
           </div>
         </div>
       </div>
     </div>
   </div>
-  <div class="container-fluid">
-    <div class="hidden md:grid md:gap-6 md:grid-cols-3">
-      <a href="#beginner_section" class="py-3 lg:py-6 flex items-center gap-2 text-tertiary font-bold hover:underline">
-        <%s! Learn_components.beginner_section_icon %>
-        <span class="hidden lg:inline">Jump to</span> Beginner Section
-        <%s! Learn_components.little_down_arrow_icon %>
-      </a>
-      <a href="#intermediate_section" class="py-3 lg:py-6 flex items-center gap-2 text-[#3882B7] font-bold hover:underline">
-        <%s! Learn_components.intermediate_section_icon %>
-        <span class="hidden lg:inline">Jump to</span> Intermediate Section
-        <%s! Learn_components.little_down_arrow_icon %>
-      </a>
-      <a href="#advanced_section" class="py-3 lg:py-6 flex items-center gap-2 text-primary font-bold hover:underline">
-        <%s! Learn_components.advanced_section_icon %>
-        <span class="hidden lg:inline">Jump to</span> Advanced Section
-        <%s! Learn_components.little_down_arrow_icon %>
-      </a>
+  <div class="w-full bg:white dark:bg-dark-background_jumping_section">
+    <div class="container-fluid bg:white dark:bg-dark-background_jumping_section">
```

Comment:
Setting background color twice is unnecessary here. Color can be set on the outer container.